controllers/articles.controller.js

```diff
@@ -7,3 +7,12 @@ exports.getArticleById = async ({ params: { article_id } }, res, next) => {
     next(err);
   }
 };
+
+exports.getArticles = async (req, res, next) => {
+  try {
+    const article = await fetchArticles();
```

Comment:
same comment as above about this being pluralised :)

Reply:
fixed :)